src/main/java/ghbot/Event.java

```diff
@@ -0,0 +1,40 @@
+package ghbot;
+
+/**
+ * Event Class.
+ * This class contains information about the event.
+ */
+public class Event extends Task {
+    private String fromTime;
+    private String toTime;
+
+    /**
+     * Event Constructor
+     * @param description Description of the event.
+     * @param fromTime Start time of the event.
+     * @param toTime End time of the event.
+     */
+    public Event(String description, String fromTime, String toTime) {
+        super(description);
+        this.fromTime = fromTime;
+        this.toTime = toTime;
+    }
+
+    /**
+     * Returns a string description for the item in the list for file operation.
+     * @return a string.
+     */
+    @Override
+    public String toFile() {
```

Comment:
Excellent use of proper method naming with the use of 'verbs' throughout the codebase 😄